diff --git a/profile_app.sh b/profile_app.sh
index d5155ca..83381ed 100755
--- a/profile_app.sh
+++ b/profile_app.sh
@@ -10,7 +10,8 @@ sudo insmod $MEMPROF_PATH/module/memprof.ko
 rm /tmp/data.raw.*
 
 echo 'b' > /proc/memprof_cntl
-env LD_PRELOAD=$MEMPROF_PATH/library/ldlib.so $@
+#env LD_PRELOAD=$MEMPROF_PATH/library/ldlib.so $@
+$@
 echo 'e' > /proc/memprof_cntl
 
 cat /proc/memprof_ibs > ibs.raw 
